quickwit/quickwit-cli/src/source.rs

```diff
@@ -574,8 +574,8 @@ mod tests {
                 json!({"foo": {"bar": JsonValue::Bool(true)}, "baz": JsonValue::Bool(false)})
             ),
             vec![
-                ("foo.bar".to_string(), JsonValue::Bool(true)),
                 ("baz".to_string(), JsonValue::Bool(false)),
+                ("foo.bar".to_string(), JsonValue::Bool(true)),
```

Comment:
what cause the change?

Reply:
ok so aws forces the `preserve_order` feature from `serde_json`... Hmmm we need to make sure this does not lead to a massive regression before shipping this.

Reply:
At first sight, it seems preserve_order is only enabled for tests. Do you happen to know a way to make sure this is the case?

Reply:
or even better, see if there is a set of optjions to prevent aws from relying on `serde_json`   at all or with `preserve_order`

Reply:
> At first sight, it seems preserve_order is only enabled for tests.

from what i can tell, this is indeed the case

> Do you happen to know a way to make sure this is the case?

no, but i can try to find one. An option (which i don't like) would be to override our dependencies with fork of `serde_json` for which that feature does nothing